src/AddIns/Uno.UI.Lottie/ts/Uno.UI.Lottie.ts

```diff
@@ -145,15 +143,15 @@ namespace Uno.UI {
 			runningAnimation.properties = newProperties;
 		}
 
-		private static createAnimation(properties: LottieAnimationProperties): RunningLottieAnimation {
+		private static createAnimation(properties: LottieAnimationProperties, animationData?: AnimationData): RunningLottieAnimation {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5477430)